spec/models/remote_setting_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe RemoteSetting do
+  describe 'validations' do
+    it 'validates that our github repo is white listed' do
+      location = 'https://raw.githubusercontent.com/18F/identity-idp/master/config/agencies.yml'
+      valid_setting = RemoteSetting.create(name: 'agencies.yml', url: location, contents:'')
```

Comment:
I think you can use `build` instead of `create` here to make the test faster.

Reply:
Excellent idea.